extension.js

```diff
@@ -466,26 +514,71 @@ const ClipboardIndicator = Lang.Class({
         let that = this;
 
         Clipboard.get_text(CLIPBOARD_TYPE, function (clipBoard, text) {
-            that._processClipboardContent(text);
+            if (text === null && SAVE_IMAGE) {
+                Clipboard.get_content(CLIPBOARD_TYPE, SUPPORT_PNG ? 'image/png' : 'image/jpeg', function (clipboard, content) {
+                    that._processClipboardContent(content.get_data(), 'image', content.hash());
+                })
+            }
+            else {
+                const itens = text.split('\n');
```

Comment:
I think `text` can be null here. Consider:

1. let `text` be null
2. let `SAVE_IMAGE` be false

In that case, we fall through to the `else` block on line 522, and this line will fail when it tries to split a null value.

Also, as an aside: should this variable (`const itens`) be named `items`?